### PR TITLE
Update BA connect cloud account with new role for consent the BA application

### PR DESCRIPTION
**Pull request by xugy99**
## What Changed?

- Add "Privileged Role Administrator" the connect your cloud to BA section for Azure.

Please check https://enterprisedb.atlassian.net/browse/UPM-2268 for more details.

In short: we tested and validated that "Privileged Role Administrator" can also be used to approve the BA application in the customer's Azure AD. 

## Checklist

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx [hunk 1]
```diff
@@ -13,8 +13,11 @@ Set up your BigAnimal account on Azure Marketplace. Your Azure subscription for
 1. Create an Azure Active Directory Application client to delegate Identity and Access Management functions to Azure Active Directory (AD). You can create the Azure Active Directory Application using the Azure Portal, but a simpler and less error-prone approach is to use the `create-spn` script (see [Create Azure Active Directory Application using `create-spn`](#create-azure-active-directory-application-using-create-spn)). The script approach requires the Azure API.
 
 !!! Note
-    Some steps of the subscription process require approval of an Azure AD global administrator.
-    Your Azure role must be global administrator or you need the cooperation of an Azure AD global administrator in your organization.
+    Some steps of the subscription process require approval of an Azure AD administrator.
+    Your Azure role must be the below either of the below roles in the Azure AD
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx:17]:
```suggestion
    Your Azure role in the Azure AD must be either:
```

File: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx [hunk 2]
```diff
@@ -13,8 +13,11 @@ Set up your BigAnimal account on Azure Marketplace. Your Azure subscription for
...
+      - Global Administrator
+      - Privileged Role Administrator
+    or you need the cooperation of an user with the above roles in your organization.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx:20]:
```suggestion
    or you need the cooperation of a user with the roles in your organization.
```

File: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx [hunk 3]
```diff
@@ -100,7 +103,7 @@ Connect your cloud account with your Azure subscription.
 
 #### 4. Configure your account.
 !!! Note
-    After step 1, you are prompted for approval by an Azure AD global administrator.
+    After step 1, you are prompted for approval by an Azure AD administrator. Your Azure AD's `Global Administrator` or `Privileged Role Administrator` can approve it.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx:106]:
```suggestion
    After step 1, you are prompted for approval by an Azure AD administrator with either the Global Administrator or Privileged Role Administrator role.
```

**Comment by xugy99** [comment]
Accepted all the suggestions arleady. @drothery-edb

**Review by nataliawojcik27 — APPROVED** [review]

**Pull request merged**